pinot-core/src/test/java/org/apache/pinot/core/operator/filter/NotFilterOperatorTest.java

```diff
@@ -35,7 +36,7 @@ public class NotFilterOperatorTest {
   @Test
   public void testNotOperator() {
     int[] docIds1 = new int[]{2, 3, 10, 15, 16, 17, 18, 21, 22, 23, 24, 26, 28};
-    Set<Integer> expectedResult = new HashSet();
+    Set<Integer> expectedResult = new LinkedHashSet();
     expectedResult.addAll(Arrays.asList(0, 1, 4, 5, 6, 7, 8, 9, 11, 12, 13, 14, 19, 20, 25, 27, 29));
```

Comment:
```suggestion
    List<Integer> expectedResult = Arrays.asList(0, 1, 4, 5, 6, 7, 8, 9, 11, 12, 13, 14, 19, 20, 25, 27, 29);
```

Reply:
Thanks for the prompt reply! I have changed the patch based on the suggestion.